### PR TITLE
Update network watcher workbook name

### PR DESCRIPTION
**Pull request by morshabi**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by morshabi** [comment]
@preetikr can you please review the changes?

**Comment by preetikr** [comment]
Yes, I'll review @morshabi - assigned to myself

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -306,7 +306,7 @@
     "dataConnectorsDependencies": [],
     "previewImagesFileNames": [ "AzureNetworkWatcherWhite.png", "AzureNetworkWatcherBlack.png" ],
     "version": "1.1",
-    "title": "Azure Network Watcher",
+    "title": "Network Watcher Traffic Analtics",
```
Comment (Amitbergman) [inline Workbooks/WorkbooksMetadata.json:309]: typo - analytics?

Comment (preetikr) [reply Workbooks/WorkbooksMetadata.json:309]: Hi @liatlishams - please look into this and if still applicable fix / approve / merge and if not applicable please close the PR. This is an old one. Thanks.

**Pull request closed without merging**